front/pages/poke/[wId]/assistants/[aId]/index.tsx

```diff
@@ -51,63 +52,61 @@ const DataSourcePage = ({
+                key={a.version}
+                title={`@${a.name} (${a.sId}) v${a.version}`}
+                visual={<></>}
+              >
+                <ContextItem.Description>
+                  <div className="flex flex-col gap-2">
+                    <div className="ml-4 pt-2 text-sm text-element-700">
+                      <div>createdAt: {`${a.versionCreatedAt}`}</div>
+                      <div>
+                        scope: <b>{a.scope}</b>
                       </div>
-                      <div className="pt-2 text-sm text-element-700">
-                        model:{" "}
-                        {`${JSON.stringify(a.generation?.model, null, 2)}`}
+                      <div>versionAuthorId: {a.versionAuthorId}</div>
+                    </div>
+                    <div className="ml-4 text-sm text-element-700">
+                      <div className="font-bold">Instructions:</div>
+                      {a.generation?.prompt}
+                    </div>
+                    <div className="ml-4 text-sm text-element-700">
+                      <div className="font-bold">model:</div>
+                      {`${JSON.stringify(a.generation?.model, null, 2)}`}
+                    </div>
+                    {a.action && isRetrievalConfiguration(a.action) && (
+                      <div className="mb-2 ml-4 flex-col text-sm text-gray-600">
+                        <div className="font-bold">Data Sources:</div>
+                        {JSON.stringify(a.action.dataSources, null, 2)}
                       </div>
-                      {a.action && isRetrievalConfiguration(a.action) && (
-                        <div className="mb-2 flex-col text-sm text-gray-600">
-                          <div className="font-bold">Data Sources:</div>
-                          {JSON.stringify(a.action.dataSources, null, 2)}
+                    )}
+                    {a.action && isDustAppRunConfiguration(a.action) && (
+                      <div className="mb-2 ml-4 flex-col text-sm text-gray-600">
+                        <div className="font-bold">Dust app:</div>
+                        <div>
+                          {a.action.appWorkspaceId}/{a.action.appId}
                         </div>
-                      )}
-                      {a.action && isDustAppRunConfiguration(a.action) && (
-                        <div className="mb-2 flex-col text-sm text-gray-600">
-                          <div className="font-bold">Dust app:</div>
-                          <div>
-                            {a.action.appWorkspaceId}/{a.action.appId}
+                      </div>
+                    )}
+                    {a.action && isTablesQueryConfiguration(a.action) && (
+                      <div className="mb-2 ml-4 flex-col text-sm text-gray-600">
+                        <div className="font-bold">Tables:</div>
+                        {a.action.tables.map((t) => (
+                          <div key={t.tableId}>
```

Comment:
nit: key here must be `{t.workspaceId}/{t.dataSourceId}/{t.tableId}` otherwise it could collide. Feel free to use `tableKey` helper